pandas/core/accessor.py

```diff
@@ -5,15 +5,17 @@
 that can be mixed into or pinned onto other pandas classes.
 
 """
+from typing import FrozenSet, Set
 import warnings
 
 from pandas.util._decorators import Appender
 
 
 class DirNamesMixin:
-    _accessors = frozenset()
+    _accessors = set()  # type: Set[str]
     _deprecations = frozenset(
-        ['asobject', 'base', 'data', 'flags', 'itemsize', 'strides'])
+        ['asobject', 'base', 'data', 'flags', 'itemsize', 'strides']
+    )  # type: FrozenSet[str]
```

Comment:
Is this annotation required? I figured it should be able to infer from data and didn't see it listed in the linked issue

Reply:
No it's not required, but `_deprecations` is typed everywhere(generic.py and frames.py) so I typed it here too.

Reply:
Yea I understand your reasoning but I think would rather remove. Should leverage inference where correct and leave annotations to where required or inference is wrong.

Reply:
@WillAyd I will remove it hold on a minute.